code/pizzabot.js

```diff
@@ -1,7 +1,61 @@
 const vegetarian = "Vegetarian Pizza"
 const hawaiian = "Hawaiian Pizza"
 const pepperoni = "Pepperoni Pizza"
-
 const pizzaPrice = 80
 
-//Put your Javscript code here:
+console.log(vegetarian);
+console.log(hawaiian);
+console.log(pepperoni);
+console.log(pizzaPrice);
+
+const buttonClicked = () => {
+alert(`Hey! Happy to serve your pizza. On our menu we have ${vegetarian}, ${hawaiian} and ${pepperoni}.`);
+orderName = prompt(`Enter the name of the pizza you want to order today.`);
+if (checkOrderName(orderName)) {
+  totalCost();
+  cookingTime();
+  orderUp();
+}
+}
+
+const checkOrderName = () => {
+  if (orderName == vegetarian || orderName == hawaiian || orderName == pepperoni) {
+  return true;
+  } else {
+    alert("Sorry, we don't have that on the menu. Please try again.");
+    buttonClicked();
+    return false;
+  }
+}
+
+const totalCost = (orderQuantity) => {
+  var element = document.getElementById("money-feedback");
```

Comment:
Try to use `const` or `let` before using `var` where it's possible!
```suggestion
  const element = document.getElementById("money-feedback");
```